components/services/bat_ledger/public/cpp/ledger_client_mojo_proxy.h

```diff
@@ -249,6 +249,10 @@ class LedgerClientMojoProxy : public mojom::BatLedgerClient,
       const std::vector<std::string>& id_list,
       DeleteUnblindedTokenCallback callback) override;
 
+  void DeleteUnblindedTokensForPromotion(
+    const std::string& promotion_id,
```

Comment:
should be 4 spaces

Reply:
Ah yes, sorry - I'd subconsciously followed the existing code - will fix.

Reply:
Done